### ISSUE TITLE
`undefined symbol: jl_typeinf_begin` on 1.9

### ISSUE THREAD

**Issue by KristofferC** [JuliaCompilerPlugins/CodeInfoTools.jl#20]
Happens when testing the package on 1.9, pkgeval log: https://s3.amazonaws.com/julialang-reports/nanosoldier/pkgeval/by_hash/c6440b9_vs_d0156b5/CodeInfoTools.primary.log

**Comment by femtomc** [comment]
and that's what happens when you use `Core.Compiler` kiddos ... :)

**Comment by femtomc** [comment]
Right, this is the experimental stuff. @KDr2 are you using the `code_inferred` interface from this package? If yes, we should figure out how type inference has changed and correct our interface.

**Comment by KDr2** [comment]
> Right, this is the experimental stuff. @KDr2 are you using the `code_inferred` interface from this package? If yes, we should figure out how type inference has changed and correct our interface.

I am not using it now, I've turned to `Core.Compiler` to obtain the type info, so any change is fine to me :)
